docs/automated-tests/recording/overview.md

```diff
@@ -1,14 +1,16 @@
 ---
 title: How to Record Test
 page_title: How to Record Test
-description: "Test Studio test Recording codeless. Record a test in Test Studio without writing code. Launch new browser to record a test. Can I start recording from certain point in my application. Can I attach Test Studio to a running browser or application. Codeless test automation with Test Studio."
-position: 0
+description: "Test Studio allows you to record tests against an application without writing code. You can attach Test Studio to a running browser or application."
+position: 0"
 ---
 # How to Record Test
 
-The **Test Recording** is one of the leading Test Studio features, which automatically captures your actions against the application under test. While recording, you can also add various verification steps to ensure the page state. A key benefit to mention are the built-in translators for all Telerik components, which allow you to automate specific actions and verifications for the controls. And all these are part of the recording flow without the need of a single line of code.
```

Comment:
Too much adjectives and marketing. Get straight to the point, readers are in a hurry.